modules/org.restlet.ext.apispark/src/org/restlet/ext/apispark/internal/model/Representation.java

```diff
@@ -114,4 +112,37 @@ public void setRaw(boolean raw) {
     public void setSections(List<String> sections) {
         this.sections = sections;
     }
+
+    public void addSections(Collection<String> sections) {
+        for (String section : sections) {
```

Comment:
in swagger core, `sections` field is null by default, and `add...` method initializes it with an empty list if needed.
I think that pattern is good